reactor-test/src/main/java/reactor/test/util/RaceTestUtils.java

```diff
@@ -100,56 +102,65 @@
 	}
 
 	/**
-	 * Synchronizes the execution of two {@link Runnable} as much as possible
+	 * Synchronizes the execution of several {@link Runnable}s as much as possible
+	 * to test race conditions. The method blocks until all have run to completion.
+	 * @param rs the runnables to execute
+	 */
+	public static void race(final Runnable... rs) {
+		race(Schedulers.elastic(), rs);
+	}
+
+	/**
+	 * Synchronizes the execution of two {@link Runnable}s as much as possible
 	 * to test race conditions. The method blocks until both have run to completion.
-	 * @param r1 the first runnable
-	 * @param r2 the second runnable
+	 * Kept for binary compatibility, see the varargs variant.
+	 * @param r1 the first runnable to execute
+	 * @param r2 the second runnable to execute
 	 */
 	public static void race(final Runnable r1, final Runnable r2) {
-		race(r1, r2, Schedulers.single());
+		race(new Runnable[]{r1, r2});
 	}
 
 	/**
 	 * Synchronizes the execution of two {@link Runnable} as much as possible
 	 * to test race conditions. The method blocks until both have run to completion.
+	 * @param s the {@link Scheduler} on which to execute the runnables
 	 * @param r1 the first runnable
 	 * @param r2 the second runnable
-	 * @param s the {@link Scheduler} on which to execute the runnables
+	 * @deprecated use {@link #race(Scheduler, Runnable...)}
```

Comment:
our deprecation policy mandates that we indicate in the deprecated comment the earliest version at which this can be removed, namely `3.6.0`